Resources/templates/CommonAdmin/ActionsAction/ActionsBuilderAction.php.twig

```diff
@@ -24,6 +24,11 @@ class ActionsController extends BaseController
      */
     public function objectAction(Request $request, $pk, $action)
     {
+        // this is needed for confirmed object actions
+        if ($action == 'edit' || $action == 'show') {
+            return $this->redirectToRoute('{{ builder.baseActionsRoute }}_'.$action, array('pk' => $pk));
+        }
+
```

Comment:
I think this is not necessary as edit / show actions are not mapped to object actions. The right routes should match first.

To be sure, you can also update your `show` modal handlers to dynamically update the $(form).prop('method') to get or post depending on protected or not action. That would be better.